src/mod/endpoints/mod_sofia/mod_sofia.c

```diff
@@ -4775,8 +4860,6 @@ static switch_call_cause_t sofia_outgoing_channel(switch_core_session_t *session
 			goto error;
 		}
 
-		profile = gateway_ptr->profile;
```

Comment:
this change along with the one at the bottom reverts a recent change i made to fix a leak here.  Why are you doing this?